packages/react-styled-core/src/Checkbox/index.js

```diff
@@ -0,0 +1,98 @@
+import React, { forwardRef } from 'react';
+import Box from '../Box';
+import ControlBox from '../ControlBox';
+import Icon from '../Icon';
+import useColorMode from '../useColorMode';
+import VisuallyHidden from '../VisuallyHidden';
+import checkboxStyles from './styles';
+
+const Checkbox = forwardRef(
+  (
+    {
+      id,
+      name,
+      value,
+      'aria-label': ariaLabel,
+      'aria-labelledby': ariaLabelledBy,
+
+      defaultIsChecked,
+      checked,
+      disabled,
+      readOnly,
+      isIndeterminate,
+      isInvalid,
+
+      variantColor = 'blue',
+      size = 'md',
+      iconColor,
+      iconSize = '12px',
+
+      onChange,
+      onBlur,
+      onFocus,
+
+      children,
+      ...rest
+    },
+    ref,
+  ) => {
+    const { colorMode } = useColorMode();
+    const styleProps = checkboxStyles({ color: variantColor, size, colorMode, isIndeterminate });
+    const opacity = readOnly || disabled ? 0.32 : 1;
+    const _defaultIsChecked = defaultIsChecked ? undefined : checked;
```

Comment:
Rename `defaultIsChecked` to `defaultChecked`

Reply:
Done!